### PR TITLE
Feature/imageupload

### PR DESCRIPTION
**Pull request by pusztaienike**

### CODE REVIEW

File: examples/sn-react-imageupload/public/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta name="description" content="Sensenet imageupload example app" />
+    <title>Sensenet imageupload example app</title>
```
Comment (herflis) [inline examples/sn-react-imageupload/public/index.html]:
```suggestion
    <title>sensenet imageupload example app</title>
```

**Comment by B3zo0** [comment]
This example not done. https://github.com/sn-panda/snimageupload/pull/9 this pr is not merged in and the master is empty.

File: examples/sn-react-imageupload/test/app.test.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+      )
+    })
+    expect(repository.loadCollection).toBeCalledTimes(1)
+    await act(async () => {
+      wrapper.find(SimpleAppBar).prop('uploadsetdata')()
+    })
+    expect(repository.loadCollection).toBeCalledTimes(2)
+  })
+})
+describe('Notification Controll', () => {
```
Comment (herflis) [inline examples/sn-react-imageupload/test/app.test.tsx]:
```suggestion
describe('Notification Control', () => {
```

File: examples/sn-react-imageupload/test/login-form.test.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+describe('LoginForm', () => {
+  const testprop = {
+    onLogin: jest.fn(),
+    error: 'test',
+  }
+  it('LoginForm snapshot', () => {
+    const wrapper = shallow(<LoginForm {...testprop} />)
+    expect(wrapper).toMatchSnapshot()
+  })
+  it('Username change test', async () => {
```
Comment (herflis) [inline examples/sn-react-imageupload/test/login-form.test.tsx]:
```suggestion
  it('should change the username', async () => {
```

File: examples/sn-react-imageupload/test/login-form.test.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    })
+    expect(
+      wrapper
+        .update()
+        .find(TextField)
+        .first()
+        .props().value,
+    ).toEqual('testusername')
+  })
+  it('Repository change test', async () => {
```
Comment (herflis) [inline examples/sn-react-imageupload/test/login-form.test.tsx]:
```suggestion
  it('should change the repository', async () => {
```

File: examples/sn-react-imageupload/package.json [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    "build": "react-scripts build",
+    "eject": "react-scripts eject"
+  },
+  "devDependencies": {
+    "@types/jest": "^24.0.23",
+    "@types/node": "^12.12.14",
+    "@types/react": "^16.9.13",
+    "@types/react-dom": "^16.9.4",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
```
Comment (B3zo0) [inline examples/sn-react-imageupload/package.json]:
```suggestion
```

File: examples/sn-react-imageupload/package.json [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "eject": "react-scripts eject"
+  },
+  "devDependencies": {
+    "@types/jest": "^24.0.23",
+    "@types/node": "^12.12.14",
+    "@types/react": "^16.9.13",
+    "@types/react-dom": "^16.9.4",
+    "react": "^16.12.0",
```
Comment (B3zo0) [inline examples/sn-react-imageupload/package.json]:
```suggestion
```

File: examples/sn-react-imageupload/package.json [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+    "react-dom": "^16.12.0",
+    "react-scripts": "^3.2.0",
+    "typescript": "^3.7.2"
+  },
+  "dependencies": {
+    "@material-ui/core": "^4.6.1",
```
Comment (B3zo0) [inline examples/sn-react-imageupload/package.json]: @material-ui/icons package is missing.

File: examples/sn-react-imageupload/test/repository-provider.test.tsx [hunk 8]
```diff
@@ -0,0 +1,14 @@
+import { shallow } from 'enzyme'
+import React from 'react'
+import { RepositoryProvider } from '../src/context/repository-provider'
+
+describe('Repository Context Provider', () => {
+  it('Matches snapshot', () => {
```
Comment (B3zo0) [inline examples/sn-react-imageupload/test/repository-provider.test.tsx]: Please remove this file.

File: examples/sn-react-browser/build/asset-manifest.json [hunk 9]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (B3zo0) [inline examples/sn-react-browser/build/asset-manifest.json]: I'm really not sure how you managed to commit these files. Please remove them.

**Pull request merged**